src/components/Autocomplete/Autocomplete.stories.tsx

```diff
@@ -0,0 +1,207 @@
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { useState } from "react";
+import { Meta, Story } from "@storybook/react";
+
+import Autocomplete from "./Autocomplete";
+import { AutocompleteProps } from "./Autocomplete.types";
+
+import StoryThemeProvider from "../../utils/StoryThemeProvider";
+import GlobalStyles from "../GlobalStyles/GlobalStyles";
+import FormLayout from "../FormLayout/FormLayout";
+import DownloadIcon from "../Icons/DownloadIcon";
+import UploadIcon from "../Icons/UploadIcon";
+import UsersIcon from "../Icons/UsersIcon";
+import { SelectorType } from "../../global/global.types";
+
+export default {
+  title: "MDS/Forms/Autocomplete",
+  component: Autocomplete,
+  argTypes: {},
+} as Meta<typeof Autocomplete>;
+
+const Template: Story<AutocompleteProps> = ({
+  label,
+  required,
+  tooltip,
+  disabled,
+  options,
+  placeholder,
+  displayDropArrow,
+}) => {
+  const [AutocompleteedValue, setAutocompleteedValue] = useState<string>(
+    placeholder ? "" : "value1",
+  );
+
+  let useOpts: SelectorType[] = [
+    { label: "Option 1", value: "value1" },
+    { label: "Option 2", value: "value2" },
+    {
+      label: "Option 3",
+      value: "value3",
+    },
+    {
+      label: "Disabled Option",
+      value: "value4",
+      disabled: true,
+    },
+  ];
+
+  if (options && options.length > 0) {
+    useOpts = options;
+  }
+
+  return (
+    <StoryThemeProvider>
+      <GlobalStyles />
+      <FormLayout>
+        <Autocomplete
+          id={"story-Autocomplete"}
+          options={useOpts}
+          value={AutocompleteedValue}
+          onChange={(newValue, extraValue) => {
+            setAutocompleteedValue(newValue);
+
+            console.log(extraValue);
+            if (extraValue) {
+              alert(`Extra Value ${JSON.stringify(extraValue)}`);
+            }
+          }}
+          label={label}
+          required={required}
+          tooltip={tooltip}
+          disabled={disabled}
+          placeholder={placeholder}
+          displayDropArrow={displayDropArrow}
+        />
+      </FormLayout>
+    </StoryThemeProvider>
+  );
+};
+
+export const Default = Template.bind({});
+Default.args = {
+  label: "A Autocomplete box",
+};
+
+export const RequiredTag = Template.bind({});
+RequiredTag.args = {
+  label: "A Autocomplete box",
+  required: true,
+};
+
+export const WithTooltip = Template.bind({});
+WithTooltip.args = {
+  label: "A Autocomplete box",
```

Comment:
Very small typo here, can we change this to `An Autocomplete box`?

```suggestion
  label: "An Autocomplete box",
```